sys/include/vfs_util.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    sys_vfs_util    VFS helper functions
+ * @ingroup     sys_vfs
```

Comment:
While this is the right place from the module point of view, it's hard to discover. Maybe add a pointer to the `vfs_read` doc saying that "To read a complete file into a buffer, see also @ref vfs_file_to_buffer", or merely a "@see vfs_file_to_buffer".